flashchips.c

```diff
@@ -17499,6 +17499,38 @@ const struct flashchip flashchips[] = {
 		.voltage	= {3000, 3600},
 	},
 
+	{
+		.vendor		= "XTX",
+		.name		= "XTX25F",
```

Comment:
This needs to be more specific since there are many XT25F parts. Perhaps "XT25F01" would be a better name.

Reply:
Thank you for your request.
I think so, it has to named "XT25F01". I have updated the name.
And I have the datasheet. If you didn't receive it yet, I can send you the datasheet  I received it from the XTX vendor after asking the vendor.